009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,449 @@
+%Below is a description of the key pieces of information stored in the agent's memory:
+Below is a description of all the elements of information stored in the agent's memory:
+
+\begin{itemize}
+
+\item  \textbf{orderPrevFloors}: A chronologically ordered list of the floors that the agent has been in. 
+\item  \textbf{prevFloors}: A map of all the floors that the agent has experienced. This keeps track of the number of days spent on each given floor and the average food that was available whilst the agent was on that floor.
+\item  \textbf{currentDayonFloor}:The number of days the agent has spent on the current floor.
+\item  \textbf{currentFloorRisk}: This is the level of risk to the survival of the agent associated with the current floor. 
+
+\item  \textbf{daysHungry}: The number of consecutive days the agent has not consumed food.
+\item  \textbf{seenPlatform}: This variable is used to remember if the agent has seen the platform. The agent uses this to decide when to prepare for a new day.
+\item  \textbf{prevAge}: The age of the agent on the previous day.
+\item  \textbf{prevHP}: A record of the HP at the end of the previous day.
+\item  \textbf{foodEaten}: The amount of food consumed the previous day.
+\item  \textbf{receivedReq}: This is a variable that indicates whether the agent has decided to abide by a request made in a message. A false value indicates that no requests have been accepted. 
+\item  \textbf{takeFood}: This stores the amount of food an agent has requested to be taken. It is initialised with a value of -1 if there are no accepted requests.
+\item  \textbf{leaveFood}: This stores the amount of food an agent has requested to be left. It is also initialised with a value of -1 if there are no accepted requests.
+
+\end{itemize}
+
+\subsection{Floor Risk Estimation}
+\label{subsec: Floor Map}
+
+Upon a floor reshuffle, the agent uses its experiences from past floors in order to build an expectation of the level of risk it associates with the new floor that it has been assigned to. The first step to doing this is to generate a prediction for the amount of food that will be available on the current floor. 
+
+In order to make this estimate, the agent utilises the prevFloors map from its memory. This contains information on the average food that was available on each floor that the agent has already been on. The agent approximates the relationship between the floor level and the average food available on that floor. For example let us take an agent that has been on floors 10 and 6, and is now assigned to floor 8. The agent will calculate the linear function which corresponds to the average food on floors 6 to 10. It will then use this function to obtain the estimate for the food it expects on floor 8. 
```

Comment:
```suggestion
In order to make this estimate, the agent utilises the prevFloors map from its memory. This contains information on the average food that was available on each floor that the agent has already been on. The agent approximates the relationship between the floor level and the average food available on that floor. For example, let us take an agent that has been on floors 10 and 6, and is now assigned to floor 8. The agent will calculate the linear function which corresponds to the average food on floors 6 to 10. It will then use this function to obtain the estimate for the food it expects on floor 8. 
```